syntax = "proto3";

package calculator;

service Calculator {
  rpc CalculateSum (SumRequest) returns (SumReply) {}
}

message SumRequest {
  int32 first_num = 1;
  int32 sec_num = 2;
}

message SumReply {
  int32 sum = 1;
}

